tests/test_spend_bundle.py

```diff
@@ -241,4 +241,4 @@ def test_derive_class():
     assert test.test_bool
     assert test.new_function()
 
-    test = NewAndImprovedSpendBundle.parse_rust(obj_bytes)
+    test = NewAndImprovedSpendBundle.parse_rust(obj_bytes)[0]
```

Comment:
perhaps this line should be `assert NewAndImprovedSpendBundle.parse_rust(obj_bytes)[0] is not None` ?

Reply:
`parse_rust()` doesn't return an `Optional[]` though, so that would be a type error